<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecipeMain</class>
 <widget class="QMainWindow" name="RecipeMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>852</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>393</width>
    <height>852</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>393</width>
    <height>852</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="MainMenu">
   <property name="minimumSize">
    <size>
     <width>393</width>
     <height>852</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>393</width>
     <height>852</height>
    </size>
   </property>
   <widget class="QStackedWidget" name="RecipeList">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>394</width>
      <height>851</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="homeButton">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>61</y>
        <width>37</width>
        <height>37</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/backBttn.png);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="Recipes">
      <property name="geometry">
       <rect>
        <x>68</x>
        <y>58</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Recipes</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addButton">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>730</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/Group 22.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="recipeList" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>128</y>
        <width>391</width>
        <height>673</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background:none;
background-color:transparent;</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <property name="spacing">
        <number>5</number>
       </property>
       <item row="0" column="0">
        <widget class="QScrollArea" name="scrollArea">
         <property name="styleSheet">
          <string notr="true">QScrollBar:vertical {
                border: none;
                background: #f0f0f0;
                width: 8px;
                margin: 10px 0 10px 0;
                border-radius: 4px;
            }
            QScrollBar::handle:vertical {
                background: #a0a0a0;
                min-height: 20px;
                border-radius: 4px;
            }
            QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
                background: none;
            }
QScrollArea {
    border: none;
}

QScrollBar:horizontal {
    border: none;
    background: none;
    height: 0;
}
</string>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>371</width>
            <height>653</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QWidget" name="recipeDisplay" native="true">
             <layout class="QGridLayout" name="gridLayout_5"/>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>393</width>
        <height>852</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/Untitled design.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>label</zorder>
     <zorder>homeButton</zorder>
     <zorder>Recipes</zorder>
     <zorder>recipeList</zorder>
     <zorder>addButton</zorder>
    </widget>
    <widget class="QWidget" name="recipe">
     <widget class="QLabel" name="Recipes_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>58</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Recipes</string>
      </property>
     </widget>
     <widget class="QPushButton" name="backButton_2">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>61</y>
        <width>37</width>
        <height>37</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style: outset;
border-image: url(:/images/backBttn.png);
border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="photoDisplay">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>140</y>
        <width>196</width>
        <height>176</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 25px;
background: #8ad3de;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="editButton">
      <property name="geometry">
       <rect>
        <x>321</x>
        <y>63</y>
        <width>24</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/Group.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="Recipes_3">
      <property name="geometry">
       <rect>
        <x>44</x>
        <y>345</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ingredients</string>
      </property>
     </widget>
     <widget class="QLabel" name="Recipes_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>530</y>
        <width>301</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Procedure</string>
      </property>
     </widget>
     <widget class="QListWidget" name="viewIngredients">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>380</y>
        <width>311</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget {
		padding: 2px;
        border: 5px solid #D9D9D9;
        border-radius: 5px; /* Optional: to add border radius */
    }
    QScrollBar:vertical {
        background: #FFFFFF; /* Scrollbar background color */
        width: 10px; /* Width of the vertical scrollbar */
        margin: 0px 0px 0px 0px; /* Optional: adjust margins */
    }
    QScrollBar::handle:vertical {
        border-radius: 5px; /* Border radius for the scrollbar handle */
        background: #898989; /* Scrollbar handle color */
        min-height: 20px; /* Minimum height of the scrollbar handle */
    }
    QScrollBar::add-line:vertical {
        border: none; /* Optional: remove the line above the scrollbar */
        background: none;
    }
    QScrollBar::sub-line:vertical {
        border: none; /* Optional: remove the line below the scrollbar */
        background: none;
}</string>
      </property>
      <item>
       <property name="text">
        <string>fadsf</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>fdasf</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>dsaf</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ads</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>fads</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>fads</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>f</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>adsf</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>a</string>
       </property>
      </item>
     </widget>
     <widget class="QPlainTextEdit" name="viewProcedures">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>560</y>
        <width>311</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPlainTextEdit {
		padding: 2px;
        border: 5px solid #D9D9D9;
        border-radius: 5px; /* Optional: to add border radius */
    }
    QScrollBar:vertical {
        background: #FFFFFF; /* Scrollbar background color */
        width: 10px; /* Width of the vertical scrollbar */
        margin: 0px 0px 0px 0px; /* Optional: adjust margins */
    }
    QScrollBar::handle:vertical {
        border-radius: 5px; /* Border radius for the scrollbar handle */
        background: #898989; /* Scrollbar handle color */
        min-height: 20px; /* Minimum height of the scrollbar handle */
    }
    QScrollBar::add-line:vertical {
        border: none; /* Optional: remove the line above the scrollbar */
        background: none;
    }
    QScrollBar::sub-line:vertical {
        border: none; /* Optional: remove the line below the scrollbar */
        background: none;
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>393</width>
        <height>852</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/Untitled design.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>label_2</zorder>
     <zorder>Recipes_2</zorder>
     <zorder>backButton_2</zorder>
     <zorder>photoDisplay</zorder>
     <zorder>editButton</zorder>
     <zorder>Recipes_3</zorder>
     <zorder>Recipes_5</zorder>
     <zorder>viewIngredients</zorder>
     <zorder>viewProcedures</zorder>
    </widget>
    <widget class="QWidget" name="add">
     <widget class="QPushButton" name="backButton">
      <property name="geometry">
       <rect>
        <x>29</x>
        <y>49</y>
        <width>37</width>
        <height>37</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/backBttn.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>33</x>
        <y>127</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Recipe Name</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteButton">
      <property name="geometry">
       <rect>
        <x>48</x>
        <y>734</y>
        <width>114</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/Group 35.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="Recipes_4">
      <property name="geometry">
       <rect>
        <x>81</x>
        <y>29</y>
        <width>288</width>
        <height>70</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Add a new recipe</string>
      </property>
     </widget>
     <widget class="QPushButton" name="saveButton">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>734</y>
        <width>114</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
border-image: url(:/images/Group 68.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="inputRecipeName">
      <property name="geometry">
       <rect>
        <x>33</x>
        <y>150</y>
        <width>326</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">	border-radius: 10px;
	border: none;
    background-color: #d9d9d9;
    padding: 2px;</string>
      </property>
      <property name="plainText">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>33</x>
        <y>185</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ingredients</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>33</x>
        <y>420</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Procedure</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="inputProcedure">
      <property name="geometry">
       <rect>
        <x>33</x>
        <y>443</y>
        <width>326</width>
        <height>171</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPlainTextEdit{
border: 5px solid #D9D9D9;
border-radius: 8px;
padding: 2px;
}
QScrollBar:vertical {
        background: #FFFFFF; /* Scrollbar background color */
        width: 10px; /* Width of the vertical scrollbar */
        margin: 0px 0px 0px 0px; /* Optional: adjust margins */
    }
    QScrollBar::handle:vertical {
        border-radius: 5px; /* Border radius for the scrollbar handle */
        background: #898989; /* Scrollbar handle color */
        min-height: 20px; /* Minimum height of the scrollbar handle */
    }
    QScrollBar::add-line:vertical {
        border: none; /* Optional: remove the line above the scrollbar */
        background: none;
    }
    QScrollBar::sub-line:vertical {
        border: none; /* Optional: remove the line below the scrollbar */
        background: none;
}</string>
      </property>
      <property name="plainText">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>33</x>
        <y>638</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Add Picture</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addPhoto">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>660</y>
        <width>32</width>
        <height>32</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/image 9.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="addIngredient">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>210</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/Group 140.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="inputIngredient">
      <property name="geometry">
       <rect>
        <x>33</x>
        <y>209</y>
        <width>251</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">	border-radius: 10px;
	border: none;
    background-color: #d9d9d9;
    padding: 2px;</string>
      </property>
      <property name="plainText">
       <string/>
      </property>
      <property name="centerOnScroll">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QListWidget" name="listIngredient">
      <property name="geometry">
       <rect>
        <x>33</x>
        <y>249</y>
        <width>326</width>
        <height>107</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget {
        border: 5px solid #D9D9D9;
        border-radius: 5px; /* Optional: to add border radius */
    }
    QScrollBar:vertical {
        background: #FFFFFF; /* Scrollbar background color */
        width: 10px; /* Width of the vertical scrollbar */
        margin: 0px 0px 0px 0px; /* Optional: adjust margins */
    }
    QScrollBar::handle:vertical {
        border-radius: 5px; /* Border radius for the scrollbar handle */
        background: #898989; /* Scrollbar handle color */
        min-height: 20px; /* Minimum height of the scrollbar handle */
    }
    QScrollBar::add-line:vertical {
        border: none; /* Optional: remove the line above the scrollbar */
        background: none;
    }
    QScrollBar::sub-line:vertical {
        border: none; /* Optional: remove the line below the scrollbar */
        background: none;
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteIngredient">
      <property name="geometry">
       <rect>
        <x>33</x>
        <y>363</y>
        <width>326</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/Group 141.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="photoName">
      <property name="geometry">
       <rect>
        <x>33</x>
        <y>660</y>
        <width>281</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPlainTextEdit {
	border-radius: 10px;
	border: none;
    background-color: #d9d9d9;
    padding: 2px;
}

QScrollBar:vertical {
    background: #d9d9d9; /* Scrollbar background color */
        width: 10px; /* Width of the vertical scrollbar */
        margin: 0px 0px 0px 0px; /* Optional: adjust margins */
}

QScrollBar::handle:vertical {
    border-radius: 5px; /* Border radius for the scrollbar handle */
    background: #FFFFFF; /* Scrollbar handle color */
    min-height: 20px; /* Minimum height of the scrollbar handle */
}

QScrollBar::add-line:vertical {
    border: none; /* Optional: remove the line above the scrollbar */
    background: none;
}

QScrollBar::sub-line:vertical {
    border: none; /* Optional: remove the line below the scrollbar */
    background: none;
}
</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-14</y>
        <width>393</width>
        <height>866</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/Untitled design.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>label_3</zorder>
     <zorder>backButton</zorder>
     <zorder>label_9</zorder>
     <zorder>deleteButton</zorder>
     <zorder>Recipes_4</zorder>
     <zorder>saveButton</zorder>
     <zorder>inputRecipeName</zorder>
     <zorder>label_10</zorder>
     <zorder>label_11</zorder>
     <zorder>inputProcedure</zorder>
     <zorder>label_12</zorder>
     <zorder>addPhoto</zorder>
     <zorder>addIngredient</zorder>
     <zorder>inputIngredient</zorder>
     <zorder>listIngredient</zorder>
     <zorder>deleteIngredient</zorder>
     <zorder>photoName</zorder>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="assets/assets.qrc"/>
 </resources>
 <connections/>
</ui>
